python/graphstorm/gsf.py

```diff
@@ -599,6 +600,13 @@ def set_encoder(model, g, config, train_task):
                                  num_hidden_layers=config.num_layers -1,
                                  dropout=dropout,
                                  num_ffn_layers_in_gnn=config.num_ffn_layers_in_gnn)
+    elif model_encoder_type == "gatv2":
+        gnn_encoder = GATv2Encoder(h_dim=config.hidden_size,
```

Comment:
I think we need to test if the input graph is the homogeneous graph here. Check at https://github.com/awslabs/graphstorm/blob/main/python/graphstorm/gsf.py#L586